mirrord/kube/src/api/runtime.rs

```diff
@@ -45,6 +45,9 @@ pub struct RuntimeData {
     pub container_id: String,
     pub container_runtime: ContainerRuntime,
     pub container_name: String,
+
+    /// Used to check if we're running with a mesh/sidecar in `detect_mesh_sidecar`.
+    pub container_statuses: Vec<ContainerStatus>,
```

Comment:
suggestion - maybe it should be `is_mesh` then the logic for if is mesh stays in one location + it makes sense to store that specifically.